llvm/tools/llvm-dwarfdump/llvm-dwarfdump.cpp

```diff
@@ -866,7 +866,8 @@ int main(int argc, char **argv) {
     if (Verbose)
       DumpType = DIDT_All;
     else
-      DumpType = DIDT_DebugInfo;
+      // If no options are passed, verify everything but dump only debug_info.
```

Comment:
Nit: this comment is slightly ambiguous. I think you can sidestep it by using an `else if` so it looks symmetrical with the verbose dumping. 

```
if (Verbose)
  DumpType = DIDT_All;
else if(Verify)
  DumpType = DIDT_All;
else 
  DumpType = DIDT_DebugInfo;
```

or maybe even simpler:

```
if (Verbose || Verify)
  DumpType = DIDT_All;
else 
  DumpType = DIDT_All;
```

Reply:
Yup, good point! I like your second suggestion.